doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -768,24 +768,18 @@ Modem libraries
 
 * :ref:`lte_lc_readme` library:
 
-  * Removed:
-
-    * The :c:func:`lte_lc_init` function.
-      All instances of this function can be removed without any additional actions.
-    * The :c:func:`lte_lc_deinit` function.
-      Use the :c:func:`lte_lc_power_off` function instead.
-    * The :c:func:`lte_lc_init_and_connect` function.
-      Use the :c:func:`lte_lc_connect` function instead.
-    * The :c:func:`lte_lc_init_and_connect_async` function.
-      Use the :c:func:`lte_lc_connect_async` function instead.
-    * The ``CONFIG_LTE_NETWORK_USE_FALLBACK`` Kconfig option.
-      Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option instead.
-      In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
-
-  * Deprecated the :c:macro:`LTE_LC_ON_CFUN` macro.
-    Use the :c:macro:`NRF_MODEM_LIB_ON_CFUN` macro instead.
+  * Added:
 
-  * Added a new :c:enum:`LTE_LC_EVT_RAI_UPDATE` event that is enabled with the :kconfig:option:`CONFIG_LTE_RAI_REQ` Kconfig option.
+    * The new :kconfig:option:`CONFIG_LTE_LC_CONN_EVAL` Kconfig option to enable the Connection Parameters Evaluation module.
+    * The new :kconfig:option:`CONFIG_LTE_LC_EDRX` Kconfig option to enable the eDRX module.
+    * The new :kconfig:option:`CONFIG_LTE_LC_NEIGHBOR_CELL_MEAS` Kconfig option to enable the Neighboring Cell Measurements module.
+    * The new :kconfig:option:`CONFIG_LTE_LC_PERIODIC_SEARCH` Kconfig option to enable the Periodic Search Configuration module.
```

Comment:
`CONFIG_LTE_LC_PERIODIC_SEARCH_CONTROL` would be a more intuitive name, if it is not too late to change it.

But I understand completely it is pretty late in the review process, so apply this advice at your discretion

Reply:
I don't think it's too late to change if that's what we want. The API has functions `lte_lc_periodic_search_X()` which backs the current naming `CONFIG_LTE_LC_PERIODIC_SEARCH`.
AT command that the module uses is `AT%PERIODICSEARCHCONF`.
So I don't feel `CONFIG_LTE_LC_PERIODIC_SEARCH` is too bad but I'm more or less blind and biased to this problems of this name, if problems exist.
If I had to rename this, I would probably use `CONFIG_LTE_LC_PERIODIC_SEARCH_CONF` instead of `CONFIG_LTE_LC_PERIODIC_SEARCH_CONTROL` given the AT command name. But then we are diverging from the API function names.

Please let us know your thoughts after reading this comment.

Reply:
I do like it better with a suffix, personally. I suggested `CONTROL`/`CTRL` since there is `lte_lc_periodic_search_request`, but `CONFIG`, `CONF`, or `CFG` would all be just fine to me too!

Basically it is for the same reason as the other comment -- `CONFIG_LTE_LC_PERIODIC_SEARCH` might be a little misleading since it sounds sort of like it controls whether `PERIODIC_SEARCH` is enabled. But really it just enables/disables the config/control functions.